### ISSUE TITLE
Starter Example

### ISSUE THREAD

**Issue by muhajirframe** [bradlc/babel-plugin-tailwind-components#12]
Hey @bradlc ,
I just made a react starter with this lib.  I was struggling integrating this with `create-react-app` previously.

Do you think it would be a good idea putting [react-tailwind-emotion-starter](https://github.com/muhajirframe/react-tailwind-emotion-starter) on readme?

**Comment by bradlc** [comment]
Hey @muhajirframe 

I think I would prefer to add instructions for Create React App itself. As of [version 2.0, released a few days ago](https://reactjs.org/blog/2018/10/01/create-react-app-v2.html), CRA supports Babel macros out-of-the-box (without the need for `react-app-rewired`), and I have verified that `tailwind.macro` works fine.

**Comment by muhajirframe** [comment]
Hi, @bradlc . Yeah. that could work too. 👍 

Last time I tried with babel macros. It takes the `tailwind.js` from  root project `/talwind.js`. But `create-react-app` can only import files within `/src`.

And then I try write the config in `.babel-plugin-macrosrc.js`. Which doesn't seems to work too .  Also, this feature is still experimental.

**Comment by bradlc** [comment]
Have you tried that with Create React App 2.0? I tried yesterday and it worked with this `babel-plugin-macros.config.js`:

```js
module.exports = {
  tailwind: {
    config: './src/tailwind.js'
  }
}
```

**Comment by bradlc** [comment]
Also it could be cool to add a section to the readme with links to related repos/packages

**Comment by muhajirframe** [comment]
> Have you tried that with Create React App 2.0? I tried yesterday and it worked with this `babel-plugin-macros.config.js`:
> 
> ```js
> module.exports = {
>   tailwind: {
>     config: './src/tailwind.js'
>   }
> }
> ```

Yeah, maybe when I tried that. It wasn't 2.0 yet. I'll give it another try.

**Comment by muhajirframe** [comment]
> Also it could be cool to add a section to the readme with links to related repos/packages

Yeah, I published some lib after using your lib
- [gatsby-tailwind-emotion-starter](https://github.com/muhajirframe/gatsby-tailwind-emotion-starter)
- [gatsby-plugin-tailwindcss](https://github.com/muhajirframe/gatsby-plugin-tailwindcss)
- [vscode-tailwind-styled-snippets](https://github.com/muhajirframe/vscode-tailwind-styled-snippets)
